tests/unit/conftest.py

```diff
@@ -94,8 +94,6 @@ def change_directory(self, new_working_directory: Path) -> 'MockPathLookup':
 
     def resolve(self, path: Path) -> Path | None:
         candidates = [path]
-        if not path.name.endswith('.txt'):
```

Comment:
this causes a lot of unrelated changes for this PR, please revert it from this one and file a separate, as it obstructs the review.

Reply:
done

Reply:
done